burnman/tools.py

```diff
@@ -643,3 +644,109 @@ def check_eos_consistency(m, P=1.e9, T=300., tol=0.01, verbose=False):
             print('Not satisfied all EoS consistency constraints for {0:s}'.format(m.to_string()))
             
     return consistency
+
+
+def _pad_ndarray_inverse_mirror(array, padding):
+    """
+    Pads an ndarray according to an inverse mirror
```

Comment:
You saw the different ``mode`` options of ``gaussian_filter``, right? I can not find their precise definition, but did you try  ``mode=refect``?